docs/EN_US/ContainerizedHPCC/ContainerizedMods/AzureDeployChapter.xml

```diff
@@ -0,0 +1,645 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="HPCC_Azure_deployment">
+  <title>Azure Deployment (Development, Testing, and Production)</title>
+
+  <para>The following section and commands are all intended to be general
```

Comment:
This section should apply to most Azure subscriptions. You may need to adjust some commands to fit your subscription's requirements.